template/src/androidLibAndroidTest/kotlin/README.md

```diff
@@ -0,0 +1,43 @@
+## Android instrumented unit tests
```

Comment:
Do we want the same README in every component?

Reply:
This readme will be out of date really quickly.

Reply:
Probably it is not a common case, but for me as for iOS developer took some it took some time to figure out how to start writing tests and why for android there are two different folders. I understand that you can read a few articles and take a look at the official documentation, but I would find it more convenient to have such quick HOWTO. 
In general, I just wrote down all questions I had trying to contribute to Kaluga.

Reply:
I'm not opposed to the READMEs, and if I want to improve them I shall make a PR myself. I'm against them being copied into every new module, where we cannot properly maintain them.

I would be for making the template folder level one deeper (e.g. `/adding-a-new-module/template`, or putting the even putting one or more markdown files  in the root of the whole project (since they are quite important)